Source/Core/Core/PowerPC/Jit64/Jit_FloatingPoint.cpp

```diff
@@ -560,6 +560,7 @@ void Jit64::fcmpx(UGeckoInstruction inst)
 {
 	INSTRUCTION_START
 	JITDISABLE(bJITFloatingPointOff);
+	FALLBACK_IF(inst.Rc);
```

Comment:
it does, cause it's actually not fcmp it's fcmpx - so by going off of flacs words, that's what the x stands for. Ignore my poor naming of my own branch.